src/flag.h

```diff
@@ -270,6 +270,7 @@ extern const flag_id flag_REDUCED_WEIGHT;
 extern const flag_id flag_RELOAD_AND_SHOOT;
 extern const flag_id flag_RELOAD_EJECT;
 extern const flag_id flag_RELOAD_ONE;
+extern const flag_id flag_resized_large;
```

Comment:
```suggestion
extern const flag_id flag_RESIZED_LARGE;
```

I think we should capitalize flag names for consistency.